frontend/CropProfitCalculator.css

```
@@ -686,3 +686,312 @@
+
+.theme-dark .mode-toggle {
+  background: #374151;
+  box-shadow: inset 0 2px 4px rgba(0, 0, 0, 0.2);
+}
+
+.toggle-btn {
+  background: transparent;
+  border: none;
+  padding: 10px 24px;
+  border-radius: 24px;
+  font-family: 'Poppins', sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)